src/app/factions/[id]/page.tsx

```
@@ -0,0 +1,165 @@
+import { Badge } from '@/components/ui/badge';
+import { Card, CardContent } from '@/components/ui/card';
+import { getFaction, getFactionIds } from '@/lib/data';
+import Image from 'next/image';
+/**
+ * @deprecated - This page should be migrated to frontend-next/src/app/factions/[id]/page.tsx
+ * This is a temporary wrapper during migration
+ */
+import { notFound } from 'next/navigation';
+import Image from 'next/image';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Image'. Consider to delete it or rename it.

'Image' is defined here:


(lint/suspicious/noRedeclare)